admin/view-translations-term.php

```diff
@@ -31,19 +34,23 @@
 
 		// Look for any existing translation in this language
 		// Take care not to propose a self link
-		$translation = 0;
+		$translation = null;
 		if ( isset( $term_id ) && ( $translation_id = $this->model->term->get_translation( $term_id, $language ) ) && $translation_id != $term_id ) {
 			$translation = get_term( $translation_id, $taxonomy );
 		}
 		if ( ! empty( $from_term_id ) && ( $translation_id = $this->model->term->get( $from_term_id, $language ) ) && ! $this->model->term->get_translation( $translation_id, $lang ) ) {
 			$translation = get_term( $translation_id, $taxonomy );
 		}
 
+		$add_link = '';
+		$link     = '';
 		if ( isset( $term_id ) ) { // Do not display the add new link in add term form ( $term_id not set !!! )
-			$link = $add_link = $this->links->new_term_translation_link( $term_id, $taxonomy, $post_type, $language );
+			$link = $this->links->new_term_translation_link( $term_id, $taxonomy, $post_type, $language );
+			$add_link = $link;
 		}
 
-		if ( $translation ) {
+		$is_existing_translation = $translation instanceof WP_Term;
```

Comment:
OK to introduce this new variable, I would however assign it a bit sooner just after https://github.com/polylang/polylang/pull/1443/files#diff-52adbff083959ccc3e35f99a6bc985248c93f6721de8577afafb12788baf697dR38-R43 and rename it to `$translation_exists`.

Reply:
Done in d5f169185a4ef482de76b4ae8c4de6a944c5d879